src/app/examples/todos/todos.component.spec.ts

```diff
@@ -124,9 +124,7 @@ describe('TodosComponent', () => {
     fixture.detectChanges();
     expect(component.newTodo).toBe('');
     expect(dispatchSpy).toHaveBeenCalledTimes(1);
-    expect(dispatchSpy).toHaveBeenCalledWith(
-      new ActionTodosAdd({ name: 'test' })
-    );
+    expect(dispatchSpy.calls.mostRecent().args[0].payload.name).toBe('test');
```

Comment:
We can also use `jasmine.objectContaining` here:

```ts
    expect(dispatchSpy).toHaveBeenCalledWith(
      jasmine.objectContaining({
        type: TodosActionTypes.ADD,
        payload: jasmine.objectContaining({ name: 'test' })
      })
    );
```

The downside of this is that we can't reuse the `ActionTodosAdd` action creator and that have to wrap our whole action with `jasmine.objectContaining`, because it's actually been called with an `ActionTodosAdd` action.